### ISSUE TITLE
[Mobile Payments] Show last4 and cardBrand on Refund Confirmation with Stripe gateway

### ISSUE THREAD

**Issue by joshheald** [woocommerce/woocommerce-ios#6181]
Split from #5976 

Awaiting either:
 * an API endpoint for fetching charge details equivalent to `/payments/charges/` from the Stripe Gateway pdfdoF-ru-p2 
 * or the addition of required fields to `Order.meta_data`. p91TBi-7wE-p2

**Comment by jaclync** [comment]
Maybe we can move this issue out of the 🇨🇦 milestone since we only support WCPay for the release? (ref pdfdoF-D1-p2)

**Comment by joshheald** [comment]
Yeah it would make sense to treat Stripe Gateway as a separate project. I'll make a new one as a bucket for issues like this one...

**Comment by joshheald** [comment]
Moved to Linear as WOOMOB-3581. We will work on it and track progress in Linear: https://linear.app/a8c/issue/WOOMOB-3581/mobile-payments-show-last4-and-cardbrand-on-refund-confirmation-with
